### ISSUE TITLE
Doesn't support http-client-0.5

### ISSUE THREAD

**Issue by phadej** [snoyberg/keter#153]
```
Network/HTTP/ReverseProxy/Rewrite.hs:131:9:
    ‘checkStatus’ is not a (visible) constructor field name
```

Versions since `1.3.1` are affected.

**Comment by tolysz** [comment]
Hi, could you try: https://github.com/snoyberg/keter/pull/152

**Comment by phadej** [comment]
@tolysz it probably works as travis is green.

I'm asking for revisions on Hackage, so I can install `keter` without relying on `stack` / manually adding constraints.

**Comment by tolysz** [comment]
@creichert if we could start looking into the PRs :) so we can help users to enjoy the convenient ways of installing `keter`...

**Comment by creichert** [comment]
@tolysz Just made an edit, can you review: https://hackage.haskell.org/package/keter-1.4.3.1/keter.cabal

**Comment by creichert** [comment]
I just uploaded a new version: https://hackage.haskell.org/package/keter-1.4.3.2

This includes @tolysz fixes in #152. I also

If I missed something, please mention it and I will fix it and upload a new version.

Thanks!
